src/DesignMyNight/Elasticsearch/QueryBuilder.php

```diff
@@ -22,6 +22,13 @@ class QueryBuilder extends BaseBuilder
 
     protected $scrollSelect;
 
+    /**
+     * All of the supported clause operators.
+     *
+     * @var array
+     */
+    public $operators = ['=', '<', '>', '<=', '>=', '!=', 'exists'];
```

Comment:
This doesn't seem to be used anywhere.  I noticed that inside QueryGrammer::compileWhereBasic it has the operators defined.  Perhaps a constant might be better?

Perhaps in this format?

```php
const ES_OPERATORS = [ '=', '<', '>', '<=', '>=', '!=', 'exists']);
```

Reply:
This overrides the $operators array in the base Illuminate\Database\Query\Builder class.

Reply:
ah I missed that